TrackParamFilter/sections/specific.txt

```diff
@@ -1778,11 +1778,19 @@ $removeparam=igshid,domain=instagram.com|threads.net
 ||linkedin.com^$removeparam=original_referer
 ||linkedin.com^$removeparam=refId
 ||linkedin.com^$removeparam=trackingId
+! Remove user survey back to LinkedIn button tracking parameter
+||linkedin.com^removeparam=trk
```

Comment:
Need steps to reproduce.

Reply:
This is happening when you click on an ad that asks you how you'd like to use premium. 

There are tracking parameters there and also upon quitting the survey prematurely